src/en/charms-deploying.md

```diff
@@ -237,7 +259,7 @@ juju retry-provisioning 3 27 5
 
 ## Considerations
 
-Charms are running without any separation, so its entirely possible for Charms
+Charms are running without any separation, so its entirely possible for charms
 to stomp all over each others configuration files and try to bind to the same
```

Comment:
First edit pass:
Charms run without any separation. We are working to have each service run in its own container. Until that work is complete you should pay careful attention to the impact multiple charms may have on configuration files and networking to make sure that added charms do not overwrite or change settings.